### PR TITLE
Add with secrets.

### PR DESCRIPTION
**Pull request by yaohaizh**

### CODE REVIEW

File: azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachineScaleSet.java [hunk 1]
```diff
@@ -1526,6 +1526,19 @@ WithManagedCreate withNewDataDiskFromImage(int imageLun,
             WithCreate withCustomData(String base64EncodedCustomData);
         }
 
+        /**
+         * The stage of the virtual machine scale set definition allowing to specify the custom data.
```
Comment (anuchandy) [inline azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachineScaleSet.java]: `allowing to specify the custom data` -> `allowing to specify the secrets`

File: azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachineScaleSet.java [hunk 2]
```diff
@@ -1526,6 +1526,19 @@ WithManagedCreate withNewDataDiskFromImage(int imageLun,
...
+         */
+        interface WithSecrets {
+            /**
+             *  Specifies set of certificates that should be installed onto the virtual machine.
+             *
+             * @param secrets the base64 encoded custom data
```
Comment (anuchandy) [inline azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachineScaleSet.java]: `@param secrets the secrets `

**Comment by yaohaizh** [comment]
For the myTest.txt, it is encoded format  from the myCert.pfx, list it as a file to make the code consistent.

File: azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachineScaleSet.java (hunk 2 above)
Comment (yaohaizh) [reply azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachineScaleSet.java]: Thanks, addressed.

File: azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachineScaleSet.java (hunk 1 above)
Comment (yaohaizh) [reply azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachineScaleSet.java]: Thanks, addressed.

**Pull request merged**